awx/ui_next/src/screens/Job/JobOutput/JobEventSkeleton.jsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import React from 'react';
+
+const JobEventSkeletonWrapper = styled.div``;
+const JobEventSkeletonLine = styled.div`
+  display: flex;
+
+  &:hover {
+    background-color: white;
+  }
+
+  &:hover div {
+    background-color: white;
+  }
+
+  &--hidden {
+    display: none;
+  }
+`;
+const JobEventSkeletonLineToggle = styled.div`
```

Comment:
Several of these styled divs are duplicated in JobEvent and JobEventSkeleton. They should probably be put in a file of their own and imported into both components that use them.